scripts/update_deepwiki_links.py

```
@@ -114,11 +127,20 @@ def main():
                         print(f"  DeepWiki not found.")
 
-                # We update if deepwiki found, or codewiki is missing, or we have a placeholder to replace
-                should_update = (deepwiki_exists and not has_deepwiki) or not has_codewiki or has_placeholder
+                codewiki_exists = False
+                if not has_codewiki or has_placeholder:
+                    print(f"Checking CodeWiki for {filename} ({owner}/{repo})...")
+                    codewiki_exists = check_codewiki_exists(page, owner, repo)
+                    if codewiki_exists:
+                        print(f"  Found CodeWiki! Updating {filename}")
+                    else:
+                        print(f"  CodeWiki not found.")
```

Ruff (0.15.5)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)